closed/OpenJ9.gmk

```diff
@@ -333,11 +338,15 @@ run-preprocessors-j9 : stage-j9 \
 build-j9 : run-preprocessors-j9
 	@$(ECHO) Compiling OpenJ9 in $(OUTPUT_ROOT)/vm
 ifeq ($(OPENJ9_ENABLE_CMAKE),true)
-	(export OMR_DIR=$(OUTPUT_ROOT)/vm/omr OPENJ9_BUILD=true && \
-		cd $(OUTPUT_ROOT)/vm/build && $(MAKE) $(MAKEFLAGS) install \
+	(export OPENJ9_BUILD=true $(EXPORT_MSVS_ENV_VARS) \ 
```

Comment:
The trailing whitespace (after the `\`) must be removed to avoid breaking cmake builds.

Reply:
Spaced removed, thank you!